pkg/backup/backup_new_test.go

```diff
@@ -218,6 +218,54 @@ func TestBackupResourceFiltering(t *testing.T) {
 				"resources/persistentvolumes/cluster/bar.json",
 			},
 		},
+		{
+			name: "resources with velero.io/exclude-from-backup=true label are not included",
+			backup: defaultBackup().
+				Backup(),
+			apiResources: []*apiResource{
+				pods(
+					withLabel(newPod("foo", "bar"), "velero.io/exclude-from-backup", "true"),
+					newPod("zoo", "raz"),
+				),
+				deployments(
+					newDeployment("foo", "bar"),
+					withLabel(newDeployment("zoo", "raz"), "velero.io/exclude-from-backup", "true"),
+				),
+				pvs(
+					withLabel(newPV("bar"), "a", "b"),
+					withLabel(newPV("baz"), "velero.io/exclude-from-backup", "true"),
+				),
+			},
+			want: []string{
+				"resources/pods/namespaces/zoo/raz.json",
+				"resources/deployments.apps/namespaces/foo/bar.json",
+				"resources/persistentvolumes/cluster/bar.json",
+			},
+		},
+		{
+			name: "resources with velero.io/exclude-from-backup=true label are not included even if matching label selector",
+			backup: defaultBackup().
+				LabelSelector(&metav1.LabelSelector{MatchLabels: map[string]string{"a": "b"}}).
+				Backup(),
+			apiResources: []*apiResource{
+				pods(
+					withLabel(withLabel(newPod("foo", "bar"), "velero.io/exclude-from-backup", "true"), "a", "b"),
```

Comment:
this was a lazy way of adding multiple labels to the pod, I can create another helper that takes multiple labels that should be easier to read

Reply:
Not sure we have a place, but should go together with this: #1440

Reply:
Yeah, it would be nice to update the helper. One option that keeps it pretty concise would be to make it variadic, and just assume it goes `..., label key 1, label value 1, label key 2, label value 2, ...`. That way you don't have to use a map or anything else.

Reply:
@skriss nice, I was thinking about something like that, will look into variadic variables